packages/host/app/styles/app.css

```diff
@@ -139,6 +139,27 @@
   margin-top: var(--boxel-sp);
 }
 
+.contains-many-editor .empty {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: var(--boxel-sp-xxxs);
+
+  background: var(--boxel-light-100);
+  box-sizing: border-box;
+  cursor: pointer;
+  width: 100%;
+  height: 65px;
```

Comment:
Can this be expressed as a multiple of a boxel spacing size?

Reply:
updated https://github.com/cardstack/boxel/pull/494/commits/13faa03159b4e3eb155112329d337bc6a52c7403.